### PR TITLE
Link to proxy configuration docs from the routing scheme page

### PR DESCRIPTION
**Pull request by colinodell**
This should help developers avoid infinite redirect loops.

See https://github.com/symfony/symfony/issues/27603

### CODE REVIEW

**Review by fabiofdsantos — APPROVED** [review]

**Review by maxhelias — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]
@colinodell I like your contribution, but I reworded it slightly to make it look less like a Symfony problem and more like a user/config problem. Thanks!

File: security/force_https.rst [hunk 1]
```diff
@@ -105,3 +105,10 @@ role:
 
 It is also possible to specify using HTTPS in the routing configuration,
 see :doc:`/routing/scheme` for more details.
+
+.. note::
+
+    Forcing HTTPS while using a reverse proxy or load balancer requires a proper
+    configuration to avoid infinite redirect loops; see :doc:`/deployment/proxies`
+    for more details.
+
```
Comment (xabbuh) [inline security/force_https.rst]: this blank line should be removed, but we can do that while merging

File: security/force_https.rst [hunk 2]
```diff
@@ -105,3 +105,10 @@ role:
 
 It is also possible to specify using HTTPS in the routing configuration,
 see :doc:`/routing/scheme` for more details.
+
+.. note::
+
+    Forcing HTTPS while using a reverse proxy or load balancer requires a proper
+    configuration to avoid infinite redirect loops; see :doc:`/deployment/proxies`
```
Comment (xabbuh) [inline security/force_https.rst]: "redirection loops"?

**Review by xabbuh — APPROVED** [review]

File: security/force_https.rst (hunk 2 above)
Comment (colinodell) [reply security/force_https.rst]: Chrome uses "redirect loop" in its error message:

image

**Comment by colinodell** [comment]
Thanks @javiereguiluz! :+1:

**Comment by javiereguiluz** [comment]
Thanks Colin! This is now merged (in 2.8, which is now the oldest maintained branch)

**Pull request closed without merging**